Assets/Scripts/Networking/BatchRequest.cs

```diff
@@ -178,42 +87,30 @@ public void ExecuteBatch(Action<BatchRequest> a_successCallback, Action<BatchReq
 
 		private void ExecuteBatch()
 		{
-			if (m_status == EBatchStatus.Failed)
-			{
-				UpdateManager.Instance.WsServerCommunicationInteractor?.UnregisterBatchRequestCallbacks(m_batchID);
+			//All add requests are in, execute the batch
+			m_status = EBatchStatus.AwaitingResults;
 
-				//Something caused the batch to already fail, call the failure callback directly
-				m_executeWhenReady = false;
-				Debug.LogError($"Batch with ID {m_batchID} could not be executed because a call during its setup failed.");
-				if (m_failureCallback != null)
-				{
-					m_failureCallback.Invoke(this);
-				}
-			}
-			else if (m_outstandingCallRequests.Count == 0 && m_status == EBatchStatus.AwaitingExecutionIDs)
-			{
-				//All add requests are in, execute the batch
-				m_executeWhenReady = false;
-				m_status = EBatchStatus.AwaitingResults;
-
-				NetworkForm form = new NetworkForm();
-				form.AddField("batch_id", m_batchID);
-				form.AddField("async", m_async.ToString());
+			NetworkForm form = new NetworkForm();
+			form.AddField("country_id", SessionManager.Instance.CurrentUserTeamID);
+			form.AddField("user_id", SessionManager.Instance.CurrentSessionID);
+			form.AddField("batch_guid", m_batchGuid.ToString());
```

Comment:
The batch guid is added. and necessary to "back reference" the correct batch when the ExecuteBatch response is given, which has this guid